### ISSUE TITLE
"{{-" and "-}}" are not working

### ISSUE THREAD

**Issue by PavloMazhuha** [proninyaroslav/java-template-engine#1]
When I try to evaluate the template with "-" symbols I'm getting an error: "illegal number syntax: -"

**Comment by proninyaroslav** [comment]
Did you mean `{{-}}`? This is invalid syntax, as it's assumed to be a minus. You can use `{{"-"}}`, `{{print "-"}}` or similar to print this symbol.

**Comment by PavloMazhuha** [comment]
@proninyaroslav ,

I'm talking about this one https://pkg.go.dev/text/template#hdr-Text_and_spaces
It is used to remove whitespaces

**Comment by proninyaroslav** [comment]
As far as I remember, it appeared later and I didn't implement it.

**Comment by PavloMazhuha** [comment]
Is it possible to implement this?

**Comment by proninyaroslav** [comment]
Yes, but I don't have time to do it. Any help appreciated.

**Comment by PavloMazhuha** [comment]
Can you give me something?  Where should I look for what class and what should be changed? I'll try to take a look.

**Comment by proninyaroslav** [comment]
I think you should start with this Go code https://github.com/golang/go/blob/master/src/text/template/parse/lex.go#L426
And implement similar in java https://github.com/proninyaroslav/java-template-engine/blob/master/src/main/java/ru/proninyaroslav/template/Lexer.java

**Comment by pantinor** [comment]
Agreed this tool needs to be able to handle the dash symbols as tokens.
